elasticsearch/src/main/scala/com/stratio/crossdata/connector/elasticsearch/ElasticSearchRowConverter.scala

```diff
@@ -123,6 +124,10 @@ object ElasticSearchRowConverter {
 
   private def toTimestamp(value: Any): Timestamp = {
     value match {
+      case value : String =>
+        val dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'hh:mm:ss.SSS")
+        val parsedDate = dateFormat.parse(value)
+        new java.sql.Timestamp(parsedDate.getTime)
       case value: java.util.Date => new Timestamp(value.getTime)
     }
```

Comment:
It would be nice to add a default matching for the rest of the cases and log the error.